Automattic/wp-calypso
# Agency Dashboard Monitoring add warning to multi-site editing

Related to [#](1204408201748644-as-1204610622637463/f)

## Proposed Changes

* This change adds a warning component that displays when bulk editing notification settings

![Screenshot 2023-05-31 at 9 47 24 AM](https://github.com/Automattic/wp-calypso/assets/12895386/a4498a05-3a55-4ded-8d4a-f63278d493d7)


## Testing Instructions

* Use Jetpack Cloud live link below, or build this locally.
* You should automatically be redirected to /dashboard upon launching
* Your account should be an agency partner
* You should have multiple jetpack connected sites on your account for this to show.
* Click on the icon next to the notifications slider to open the settings for a single site, there should be no warning at the top of the modal
* Click the edit all button on the dashboard and click on Custom notification
* You should see the warning as illustrated in the image in the changes section

## Pre-merge Checklist
